README.md

```diff
@@ -8,40 +7,44 @@ distinguishes itself with the following features:
 - Totally transparent field encryption: the value will only be encrypted in the database, never in the value
 - Unit testing
 
+## Integrations
+
+The package supports the following integrations:
+
+- Laravel
+
+## Upgrading
+
+If you're upgrading from a previous version you can find some help with that in [the upgrading guide](UPGRADING.md).
+
 ## Installation
-Add `51systems/doctrine-encrypt` to your Composer manifest.
 
-```js
-{
-    "require": {
-        "51systems/doctrine-encrypt": "~5.0"
-    }
-}
+```bash
+composer require 51systems/doctrine-encrypt
 ```
 
 ## Configuration
-### Using ZF2
-Check out the doctrine-encrypt-module at https://github.com/51systems/doctrine-encrypt-module
 
-### Manually
-Add the event subscriber to your entity manager's event manager. Assuming `$em` is your configured entity manager:
+### Laravel
```

Comment:
The Laravel library isn't a hard dependency because it's in the dev dependencies. You will only use the LaravelEncryptor when you're using the Laravel framework and Laravel always comes with the encryption library installed. The dev dependency is only used in tests to test the integration. The `LaravelEncryptor` can be perfectly bundled within this core library. Splitting it up would be overkill imo.